MibTeX/src/de/mibtex/FileUtils.java

```diff
@@ -3,6 +3,8 @@
 import java.io.File;
 
 public class FileUtils {
+    public final static String LINEBREAK = "\r\n";
+
```

Comment:
Why not a system-independent call to System.lineSeparator?

Reply:
changed to System.lineSeparator